css/pages/auth.css

```diff
@@ -0,0 +1,95 @@
+main {
+  position: relative;
+}
+
+.auth__container {
+  height: auto;
+  width: 35rem;
+  padding: 2rem;
+  background-color: #f5f3f3;
+  font-size: 1.2rem;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-80%, -50%);
+}
+
+.auth__container .heading {
+  font-size: 1.8rem;
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.auth__container .text-field {
+  margin: 1rem 0;
+  position: relative;
+}
+
+.auth__container .form-input {
+  margin: 1rem 0;
+  padding: 1rem;
+  border: none;
+  outline: none;
+  width: 100%;
```

Comment:
use responsive grid-view with 12 columns for width =100% so you don't have to repeat it

Reply:
sure will try that. Thank you.